sklearn/metrics/tests/test_classification.py

```diff
@@ -2864,3 +2864,26 @@ def test_classification_metric_division_by_zero_nan_validaton(scoring):
     X, y = datasets.make_classification(random_state=0)
     classifier = DecisionTreeClassifier(max_depth=3, random_state=0).fit(X, y)
     cross_val_score(classifier, X, y, scoring=scoring, n_jobs=2, error_score="raise")
+
+
+def test_d2_log_loss():
```

Comment:
Please add a test with 3 or more classes.

Reply:
Thank you for the review! Any suggestions on how I can improve the tests overall? Like is this the right direction to add tests or we should employ a different strategy to compare the values somehow.

Reply:
Just have a look at the tests of log loss, e.g. `test_log_loss` in test_classification.py.